source/includes/bill_payments.md

```diff
@@ -0,0 +1,267 @@
+# Bill Payments
+
+## Get User's Bill Payments
+
+```shell
+curl -X GET http://api.highline.co/users/ae16efec-d1.../bill_payments \
+  -H "Authorization Bearer: AccessToken" \
+  -H "Content-Type: application/json"
+```
+
+> Response
+
+```json
+[{
+  "id": "e87f74be-c46...",
+  "enrollment_id": "3347c40d-2f9...",
+  "enrollment": {
+    "id": "3347c40d-2f9...",
+    "product_id": "22621f3b-563...",
+    "product": {
+      "id": "22621f3b-563...",
+      "company_id": "6335861a-f23...",
+      "company": {
+        "id": "6335861a-f23...",
+        "name": "Mrs. Maryam Swaniawski",
+        "legal_entity": "Ms. Susie Lubowitz",
+        "tax_id": "8a6465c0-f32...",
+        "phone": "104-769-1538",
+        "email": "XabJCOW@ocpDoBb.info",
+        "logo_url": "https://logo.clearbit.com/highline.co",
+        "created_at": 1631304119,
+        "updated_at": 1631304119
+      },
+      "colors": {
+        "primary": "",
+        "secondary": "",
+        "background": ""
+      },
+      "enrollments": null,
+      "link_keys": null,
+      "api_keys": null,
+      "type": "loan",
+      "name": "Product 1",
+      "description": "Consequatur voluptatem au...",
+      "validate_direct_deposits": true,
+      "validate_payout_allocation": true,
+      "validate_employment_active": true,
+      "max_payment_to_income_ratio": 0,
+      "created_at": 1631304120,
+      "updated_at": 1631304120
+    },
+    "payment_amount": 500,
+    "loan_amount": 200000,
+    "loan_term": 24,
+    "payment_frequency": "monthly",
+    "first_payment_date": 1638310741,
+    "application_reference_number": "12234",
+    "account_reference_number": "33333",
+    "days_until_expires": 30,
+    "required_employment_start_date": 1460845141,
+    "required_gross_income": 35000,
+    "required_net_income": 25000,
+    "fully_synced": false,
+    "status": "pending",
+    "created_at": 1631304121,
+    "updated_at": 1631304121
+  },
+  "transactions": [{
+      "id": "ef8b53a0-bfd...",
+      "bill_payment_id": "e87f74be-c46...",
+      "amount_expected": 50000,
+      "amount_paid": 0,
+      "paid": false,
+      "status": "",
+      "type": "",
+      "payment_order_id": "",
+      "created_at": 1631304126,
+      "updated_at": 1631304126
+  }],
```

Comment:
this actually doesn't belong here. all has_many resources should follow
parent/:id/children , i.e., /bill_payments/:id/transactions.
This is to make the route more performant and support paginations for has_many based relationships.